networking/eth2/src/main/java/tech/pegasys/artemis/networking/eth2/peers/Eth2Peer.java

```diff
@@ -99,20 +91,20 @@ public boolean isConnected() {
       final UnsignedLong step,
       final ResponseListener<BeaconBlock> listener) {
     return requestStream(
-        RpcMethod.BEACON_BLOCKS_BY_RANGE,
+        BeaconChainMethods.BEACON_BLOCKS_BY_RANGE,
         new BeaconBlocksByRangeRequestMessage(headBlockRoot, startSlot, count, step),
         listener);
   }
 
   private <I extends RpcRequest, O> CompletableFuture<Void> requestStream(
       final RpcMethod<I, O> method, I request, final ResponseStream.ResponseListener<O> listener) {
-    return invoke(method, request)
+    return sendRequest(method, request)
         .thenCompose(responseStream -> responseStream.expectMultipleResponses(listener));
   }
 
-  <I extends RpcRequest, O> CompletableFuture<ResponseStream<O>> invoke(
+  public <I extends RpcRequest, O> CompletableFuture<ResponseStream<O>> sendRequest(
```

Comment:
Bit unfortunate this became public but not clear how else to send an invalid message for `ErrorConditionsIntegrationTest`.